### DISCUSSION TITLE
Have you started using a folder as your inbox?

### DISCUSSION THREAD

**Discussion by austinsmoss** [BryanJenksCommunity/FAQ#91] [Q&A] [answered]
Hi Bryan-

Your content has been extremely helpful during my development of an Obsidian workflow for academic research. So, thank you very much!

During your recent "My Obsidian Kindle Notes Workflow" YT video, I happened to notice that you had a folder marked as "inbox". This is the first time I've noticed this. Have you recently started using an inbox folder or have I just missed it in the past? If you recently started, can you also please speak to why you made the switch and whether you are liking it so far?

Thanks for your time!
Austin

**Comment by tallguyjenks** [comment, marked as answer]
Hey! 

Good eye ;D The inbox folder is meant for quick capture on mobile If i want to grab something quickly and not have it in todoist. that's its only purpose. I haven't set up anything like using the iOS shortcuts workflow to make a new doc in that location like Curtis McHale's workflow because my vault is backed up to github not to a cloud service provider like cloud because of the old sync issues (not sure if they're still an issue, but i just didn't want to deal with it). :)
